packages/filesystem/src/browser/file-dialog/file-dialog-tree-filters-renderer.tsx

```diff
@@ -43,7 +43,13 @@ export class FileDialogTreeFiltersRenderer extends ReactRenderer {
         readonly fileDialogTree: FileDialogTree
     ) {
         super();
-        this.appliedFilters = { 'All Files': [], ...suppliedFilters, };
+
+        if (suppliedFilters && Object.entries(suppliedFilters).length > 0) {
+            this.appliedFilters = suppliedFilters;
+            fileDialogTree.setFilter(Object.entries(suppliedFilters)[0][1]);
+        } else {
+            this.appliedFilters = { 'All Files': [] };
+        }
```

Comment:
We may want to extract this code to a method so it is easier to extend.

Reply:
@vince-fugnitto 

1. Fixed electron 
2. extracted to function.

Please review again. Thanks